### ISSUE TITLE
Scripts are working only before pjax reload

### ISSUE THREAD

**Issue by s9umurun** [yiisoft/yii2#10821]
Hi guys,

I am submitting the form via pjax, submission and `$.pjax.reload({container:'#products-table'});` is working correclty ONLY the first tima after pjax reload all the scripts are not working anymore. I have tried also add `$(document).on('ready pjax:success', function(){` but still not working. Or after submission without pjax I am rendering the page and again the scripts are not working. What I am doing wrong? Please give your feedbacks. Thanks!

```
$(document).on('ready pjax:success', function(){
    $("form.products-input-style").on('beforeSubmit', function(e){

            var form = $(this);

            $.ajax({
                type : "POST",
                url: form.attr('action'),
                data: new FormData(this),
                processData: false,
                contentType: false,
                success  : function(response) {
                    $.pjax.reload({container:'#products-table'});
                },
                error : function(response){
                    console.log(response);
                }
            });

            return false;

        }).on('submit', function(e){
            e.preventDefault();
        });

VIEW :
===========

```
<?php Pjax::begin(['id' => 'products-table']); ?>
        <table class="table table-hover table-striped">
            <thead>
            <th>Product image</th>
            <th>Title</th>
            <th>Price</th>
            <th>Show/Hide</th>
            <th>Actions</th>
            </thead>
            <tbody>
            <?php foreach($dataProvider as $product): ?>
                <tr id="productRow-<?php echo $product->id; ?>">
                    <?php $editForm = ActiveForm::begin([
                            'id' => 'products-edit-form-'.$product->id,
                            'action' => ['products/product-edit'],
                            'enableClientValidation'=>false,
                            'method' => 'post',
                            'options' => [
                                'enctype' => 'multipart/form-data',
                                'class' => 'products-input-style',
                            ]
                    ]); ?>
                    <td>
                        <?php
                        echo $editForm->field($model, 'id')->hiddenInput(['value'=>$product->id])->label(false);
                    </td>
                    <td>
                        <span class="product-name-value">
                            <?php echo $product->productName; ?>
                        </span>
                        <span class="product-name-input" style="display:none">
                            <?= $editForm->field($model, 'productName')->textInput(['class'=>'form-control productName', 'value'=>$product->productName])->label(false); ?>
                        </span>
                    </td>
                    <td>
                        <span class="product-cost-value">
                            <?php echo "<i class=\"fa fa-eur\"></i>
                            <span>". number_format(round($product->cost, 2), 2); ?></span>
                        </span>
                        <span class="product-cost-input" style="display:none">
                            <?= $editForm->field($model, 'cost')->textInput(['class'=>'form-control productCost', 'value'=>$product->cost])->label(false); ?>
                        </span>
                    </td>
                    <td>
                        <?php if($product->available == 'Y'){ ?>
                            <input type="checkbox" name="toggle" checked="checked" class="toggle toggle-change" id="<?php echo $product->id; ?>" >
                            <label for="<?php echo $product->id; ?>"></label>

                        <?php } else{ ?>
                            <input type="checkbox" class="toggle" name="toggle toggle-change" id="<?php echo $product->id; ?>">
                            <label for="<?php echo $product->id; ?>"></label>
                        <?php } ?>
                    </td>
                    <td>
                        <ul class="products-action">
                            <li><a href="#" class="edit" id="<?php echo $product->id; ?>"><i class="fa fa-pencil fa-2x"></i></a></li>
                            <li><a href="#" class="delete" id="<?php echo $product->id; ?>"><i class="fa fa-trash fa-2x"></i></a></li>
                            <ul class="delete-confirm-div" style="display: none;">
                                <li><a href="#" class="delete-cancel" id="<?php echo $product->id; ?>">No</a></li>
                                <li><a href="#" class="delete-confirm" id="<?php echo $product->id; ?>">Yes</a></li>
                            </ul>
                        </ul>
                        <div class="save-btn" style="display: none">
                            <?= Html::submitButton('Save', ['class'=>'btn btn-success product-edit-submit', 'id'=>$product->id]); ?>
                        </div>
                    </td>
                </tr>
                <?php ActiveForm::end(); ?>
            <?php endforeach;?>
            </tbody>
        </table>
        <?php Pjax::end(); ?>
```
```

**Comment by SilverFire** [comment]
Duplicates https://github.com/yiisoft/yii2/issues/10822
